src/adapters/supabase/helpers/wallet.ts

```diff
@@ -169,7 +169,7 @@ export class Wallet extends Super {
     if (walletData.location_id === null) {
       throw new Error("Location ID is null");
     }
-    logger.debug("Enriching wallet location metadata", locationMetaData);
+    logger.debug(`Enriching wallet location metadata ${locationMetaData}`);
```

Comment:
Doesn't look right. The latter parameter is for JSON metadata. This is important for filtering/sorting the error class (the string) and then investigating the details (the metadata) in the SQL database

Reply:
Oh I see. Then, if I keep locationMetaData as a parameter, I need it to be a Partial<MetaData>, it does not match the locationMetaData type.
If I understand it well, I would need the logger method "debug" to handle locationMetaData ?

Sorry in advance for my questions, I am not well away of all the types used in the project:
The two types don't really match in this current state.

Reply:
wrap anything you pass as a second param in an object so that `locationMetaData` is a key of `{}`